fwoa-tools/src/binaryConverter.ts

```diff
@@ -101,37 +101,43 @@ async function uploadBinaryResource(itemKey: string, newData: string): Promise<G
     .promise();
 }
 
-async function retrieveBinaryIdsFromFolder(folderName: string): Promise<void> {
-  const itemKeys = getAllBinaryKeysInFolder(folderName);
-  logs.push(`${new Date().toISOString()}: Retrieved All Binary Keys from migration export output.`);
-
+async function retrieveBinaryIdsFromFolder(): Promise<void> {
```

Comment:
I would rename this function to `convertBinaryResource`.

Also add a comment describing what the function does on line 103. For devs in the future, it won't be immediately clear why we're writing one binary resource per file, so let's explain that in the comments as well.